baseline_gravity.py

```diff
@@ -294,19 +293,17 @@ def run_scene(json_path, controller, models, base_path="~/logs", num_steps=60, v
         choice = "plausible"
         conf = 1 - imp_score
     print('choice: {}'.format(choice))
-    print('choice: {}{}'.format(choice, implausible_time))
     choice = 1 if choice == 'plausible' else 0
-    controller.end_scene(rating=choice, score=choice, report=report)
+    controller.end_scene(rating=choice, score=(1 - imp_score), report=report)
 
 
-def main(fname: str):
+def main(scene_data: dict):
     #unity_app_file_path = "PATH_HERE/MCS-AI2-THOR-Unity-App-v0.4.3-linux/MCS-AI2-THOR-Unity-App-v0.4.3.x86_64"
     MCS_CONFIG_FILE_PATH = 'mcs_config.ini'  # NOTE: I ran the tests with option "size: 450". Different sizes might lead to worse results
     #raise AttributeError("Please fill out the unity app executable path and config path")
     controller = mcs.create_controller(config_file_or_dict=MCS_CONFIG_FILE_PATH)
     #controller = mcs.create_controller(unity_app_file_path, config_file_path=MCS_CONFIG_FILE_PATH)
-    # assumes all the steps are 60...can change this if this is not the case at test
-    NUMSTEPS = 200
+    NUMSTEPS = 100
```

Comment:
I changed the `NUMSTEPS` in a few of these files to match any changes to the number of steps in the Eval 4 data.